### PR TITLE
Replace travis with GH-actions

### PR DESCRIPTION
**Pull request by bendik**

### CODE REVIEW

File: .github/workflows/make-release.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          body: 'Release Notes and changelog'
+          draft: true 
+          prerelease: false
```
Comment (emilbayes) [inline .github/workflows/make-release.yml]: Perhaps these should be like today where gh releases are `prerelease` and the real release is `npm`?

**Comment by emilbayes** [comment]
Removing Travis is not correct yet. These GH actions don’t run the full quite yet and don’t have the full matrix of versions we support

**Comment by emilbayes** [comment]
Of course it is what we ultimately want :)

**Comment by bendik** [comment]
what more is missing if we've got 32 winny going?

**Comment by emilbayes** [comment]
It doesn’t run the test suite and it needs to run it on the supported versions in our Travis matrix :)

**Comment by bendik** [comment]
@emilbayes Should i remove the arm from the test workflow, due to their likelyhood of failing running certain methods?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        os: [ARM, windows-latest, ARM64, macos-latest, ubuntu-16.04]
+        node-version: ['10.x', '12.x', '14.x']
+    runs-on: ${{ matrix.os }}
+    name: Build 
+    steps:
+      - uses: actions/checkout@v2.1.0
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version}} 
+      - run: npm run fetch-libsodium
```
Comment (bendik) [inline .github/workflows/test.yml:20]: @emilbayes Do we need this for our tests?

**Comment by bendik** [comment]
Stuck at windows builds failing tests, the rest seems to be as far as i can take it for now.

@emilbayes got time to look at the windows logs, otherwise any feedback?

**Comment by SimenB** [comment]
546db064d0ab95a56c7f41a54d5956187dba1993, fwiw

**Pull request closed without merging**